features/facebookImport/src/model/entities/connected-advertiser.js

```diff
@@ -0,0 +1,40 @@
+export default class ConnectedAdvertiser {
+    constructor() {
+        this._name = null;
+        this._hasDataFileCustomAudience = false;
+        this._hasRemarketingCustomAudience = false;
+        this._hasInPersonStoreVisit = false;
+    }
+
+    get name() {
+        return this._name;
+    }
+
+    set name(name) {
+        this._name = name;
+    }
+
+    get hasDataFileCustomAudience() {
+        return this._hasDataFileCustomAudience;
+    }
+
+    set hasDataFileCustomAudience(booleanValue) {
+        this._hasDataFileCustomAudience = booleanValue;
+    }
+
+    get hasRemarketingCustomAudience() {
+        return this._hasRemarketingCustomAudience;
+    }
+
+    set hasRemarketingCustomAudience(booleanValue) {
+        this._hasRemarketingCustomAudience = booleanValue;
+    }
+
+    get hasInPersonStoreVisit() {
+        return this._hasInPersonStoreVisit;
+    }
+
+    set hasInPersonStoreVisit(booleanValue) {
```

Comment:
I see you've created getters/setters for all of them. Are they going to be actually used?

Reply:
This is for making the model serialisable, either using bubble wrap or the serialiser from https://github.com/polypoly-eu/polyPod/pull/501. This is part of the intermediary model that gets cached to disk.